sources/academy/tutorials/node_js/block_requests_puppeteer.md

```diff
@@ -5,27 +5,27 @@ sidebar_position: 16.2
 slug: /node-js/block-requests-puppeteer
 ---
 
-_Unfortunately, in the recent version of Puppeteer, request interception disables native cache and slows down the actor significantly. Therefore it is not recommended to follow examples shown in this article. Instead, use_ [_blockRequests_](/sdk/js/docs/api/puppeteer#puppeteerblockrequestspage-options-promise) _utility function from_ [_Apify SDK_](/sdk/js/)_. It works through different paths and doesn't slow down your process._
+:::caution Improve Performance: Use blockRequests`
```

Comment:
have you tried this locally? i remember docusaurus admonitions required a blank line here and before the closing `:::`, otherwise it failed to parse it properly. but maybe something changed, or it was a different kind of problem (maybe even something prettier related)

Reply:
yeah its about prettier (which we currently dont use here currently, but we are planning to adopt biome in the upcoming weeks/months)

https://docusaurus.io/docs/next/markdown-features/admonitions#usage-with-prettier

Reply:
Seems fine locally, TBH never had an issue without blank lines in admonitions
![obraz](https://github.com/apify/apify-docs/assets/92638966/5e37e2e9-2f3b-4b91-9008-b3206403587c)

Reply:
that screenshot actually shows another problem on this line i havent seen before, you got a trailing `` ` `` there :]

Reply:
let's also add the blank lines, we use them everywhere already, lets be consistent (and lets prepare for the future, as we would be dealing with this the minute we try to add some code formatting)